snake.py

```diff
@@ -15,64 +17,87 @@
 win.border(0)
 win.nodelay(1)
 
-key = KEY_RIGHT                                                    # Initializing values
+# Initializing values
+key = KEY_RIGHT
 score = 0
 
-snake = [[4,10], [4,9], [4,8]]                                     # Initial snake co-ordinates
-food = [10,20]                                                     # First food co-ordinates
+# Initial snake co-ordinates
+snake = [[4, 10], [4, 9], [4, 8]]
+# First food co-ordinates
+food = [10, 20]
 
-win.addch(food[0], food[1], '*')                                   # Prints the food
+# Prints the food
+win.addch(food[0], food[1], '*')
 
-while key != 27:                                                   # While Esc key is not pressed
+# While Esc key is not pressed
+while key != 27:
     win.border(0)
-    win.addstr(0, 2, 'Score : ' + str(score) + ' ')                # Printing 'Score' and
-    win.addstr(0, 27, ' SNAKE ')                                   # 'SNAKE' strings
-    win.timeout(150 - (len(snake)/5 + len(snake)/10)%120)          # Increases the speed of Snake as its length increases
-    
-    prevKey = key                                                  # Previous key pressed
-    event = win.getch()
-    key = key if event == -1 else event 
+    # Printing 'Score' and
+    win.addstr(0, 2, 'Score : ' + str(score) + ' ')
+    # 'SNAKE' strings
+    win.addstr(0, 27, ' SNAKE ')
+    # Increases the speed of Snake as its length increases
+    win.timeout(150 - (len(snake)/5 + len(snake)/10) % 120)
 
+    # Previous key pressed
+    prevKey = key
+    event = win.getch()
+    key = key if event == -1 else event
 
-    if key == ord(' '):                                            # If SPACE BAR is pressed, wait for another
-        key = -1                                                   # one (Pause/Resume)
+    # If SPACE BAR is pressed, wait for another
```

Comment:
As I read these comments, I think the intent was to create a multi-line comment which read "If SPACE BAR is pressed, wait for another one (Pause/Resume)